provider/clusterrosaclassic/cluster_rosa_classic_resource.go

```diff
@@ -76,6 +77,7 @@ const (
 	propertyRosaTfCommit      = tagsPrefix + "tf_commit"
 	waitTimeoutInMinutes      = 60
 	DefaultMachinePoolMessage = "This attribute is specifically applies for the default Machine Pool and becomes irrelevant once the resource is created. Any modifications to the default Machine Pool should be made through the Terraform imported Machine Pool resource. For more details, refer to [Default Machine Pool in ROSA Cluster](../guides/worker-machine-pool.md)"
+	clusterAdminUserName      = "cluster-admin"
```

Comment:
Shouldn't this constant be in ocm-common?

Reply:
fixed